src/main/webapp/WEB-INF/layouts/default.jsp

```diff
@@ -24,10 +24,24 @@
     <!-- Bootswatch flatly theme -->
     <link rel="stylesheet" href="https://bootswatch.com/flatly/bootstrap.min.css">
 
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+
+    <!-- Include all compiled plugins (below), or include individual files as needed -->
+    <script src="${contextRoot}/js/bootstrap.js"></script>
```

Comment:
Bootstrap is already loaded using CDN, see couple of lines below.

Reply:
That's true I've missed that part.